metisp/pymetis/src/pymetis/recipes/lm_img/metis_lm_img_basic_reduce.py

```diff
@@ -141,19 +117,52 @@ def process_images(self) -> [PipelineProduct]:
         a single combined frame that is used throughout the pipeline.
         """
 
-        Msg.info(self.__class__.__qualname__, f"Starting processing image attribute.")
+        Msg.info(self.__class__.__qualname__, f"Processing Images")
+
+        Msg.info(self.__class__.__qualname__, f"Loading calibration files")
 
         flat = cpl.core.Image.load(self.inputset.master_flat.frame.file, extension=0)
         bias = cpl.core.Image.load(self.inputset.master_dark.frame.file, extension=0)
         gain = cpl.core.Image.load(self.inputset.gain_map.frame.file, extension=0)
 
         Msg.info(self.__class__.__qualname__, f"Detector name = {self.detector}")
-
-        flat = self.prepare_flat(flat, bias)
+        
+        Msg.info(self.__class__.__qualname__, f"Loading raw images")
         images = self.prepare_images(self.inputset.raw.frameset, flat, bias)
+        Msg.info(self.__class__.__qualname__, f"Pretending to correct crosstalk")
+        Msg.info(self.__class__.__qualname__, f"Pretending to correct for linearity")
+
+        Msg.info(self.__class__.__qualname__, f"Subtracting Dark")
+
+        images.subtract_image(bias)
+
+        Msg.info(self.__class__.__qualname__, f"Flat fielding")
+
+        images.divide_image(flat)
+
+        Msg.info(self.__class__.__qualname__, f"Pretending to remove masked regions")
+
+        Msg.info(self.__class__.__qualname__, f"Combining Images")
         combined_image = self.combine_images(images, self.parameters["metis_lm_img_basic_reduce.stacking.method"].value)
```

Comment:
Is this recipe supposed to save a combined image? I thought `basic_reduce` works on and saves the individual exposures.  Combination comes later after background subtraction and calibration, doesn't it?